packages/tests/src.ts/index.spec.ts

```diff
@@ -23,19 +26,19 @@ describe('Hethers Tests', function () {
         if (!start) {
             // stop local-node
             console.log('Shutdown local node');
-            shell.exec('npx hedera-local stop');
+            shell.exec('hedera stop');
             return;
         }
 
         // set env variables for docker images until local-node is updated
-        process.env['NETWORK_NODE_IMAGE_TAG'] = '0.29.0-alpha.1';
-        process.env['HAVEGED_IMAGE_TAG'] = '0.29.0-alpha.1';
-        process.env['MIRROR_IMAGE_TAG'] = '0.62.0-rc1';
+        process.env['NETWORK_NODE_IMAGE_TAG'] = '0.32.0';
```

Comment:
Let's bump to the latest versions defined [here](https://github.com/hashgraph/hedera-local-node/pull/246)